src/V3AstNodes.h

```diff
@@ -1686,6 +1749,44 @@ class AstAssocSel final : public AstNodeSel {
     virtual int instrCount() const override { return widthInstrs(); }
 };
 
+class AstWildcardSel final : public AstNodeSel {
```

Comment:
As I said before, I'm not sure we need this, but we'll see what the Verilator people say about the whole thing. Let's keep it for now.